src/StatusQ/Components/StatusLetterIdenticon.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.13
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
+    id: statusLetterIdenticon
+
+    property string name
+    property int letterSize: 21
+
+    color: Theme.palette.miscColor5
```

Comment:
Went with an orange as default color.

I checked the usages of this component inside Status Desktop and in 99% of the cases we're providing a `color` explicitly. So this is a decent default.